service_config/build_images.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+# replace secrets
+if [ -z "$STARDOG_PASSWORD" ]
+then
+  echo "STARDOG_PASSWORD is not set. Check your secrets."
```

Comment:
@Perevalov I recommend stopping the script and returning with an error code if the password is not set. Otherwise a Qanary pipeline could be deployed with an empty password.

Reply:
Done

Reply:
@MindMaster98 Looks good now?

Reply:
@Perevalov The changes can be merged.